src/pages/instances/forms/DuplicateInstanceForm.tsx

```diff
@@ -0,0 +1,204 @@
+
+  const { data: storagePools = [], isLoading: storagePoolsLoading } = useQuery({
+    queryKey: [queryKeys.storage],
+    queryFn: () => fetchStoragePools(instance.project),
+  });
+
+  const { data: clusterMembers = [], isLoading: clusterMembersLoading } =
+    useQuery({
+      queryKey: [queryKeys.cluster],
+      queryFn: fetchClusterMembers,
+      enabled: isClustered,
+    });
+
+  const notifySuccess = (instanceName: string) => {
+    const newInstance = (
+      <Link to={`/ui/project/${instance.project}/instance/${instanceName}`}>
+        {instanceName}
+      </Link>
+    );
+    toastNotify.success(
+      <>
+        Created the instance <b>{newInstance}</b>.
+      </>,
+    );
+  };
+
+  const formik = useFormik<LxdInstanceDuplicate>({
+    initialValues: {
+      instanceName: instance.name + "-duplicate",
+      allowInconsistent: false,
+      instanceOnly: false,
+      stateful: false,
+    },
+    validationSchema: Yup.object().shape({
+      instanceName: instanceNameValidation(
+        instance.project,
+        controllerState,
+        instance.name,
+      ).required(),
+    }),
+    onSubmit: (values) => {
+      createInstance(
+        JSON.stringify({
+          //Does not yet consider targetClusterMember or storagePool
+          description: instance.description,
+          name: values.instanceName,
+          source: {
+            allow_inconsistent: values.allowInconsistent,
+            instance_only: values.instanceOnly,
+            source: instance.name,
+            type: "copy",
+          },
+          stateful: values.stateful,
+        }),
+        instance.project,
+      )
+        .then(() => {
+          notifySuccess(values.instanceName);
+          close();
+        })
+        .catch((e) => {
+          toastNotify.failure(
+            `Instance creation for "${values.instanceName}" failed.`,
+            e,
+          );
+          close();
+        });
+    },
+  });
+
+  return (
+    <Modal
+      close={close}
+      className="duplicate-instances-modal"
+      title="Duplicate Instance"
+      buttonRow={
+        <>
+          <Button
+            appearance="base"
+            className="u-no-margin--bottom"
+            type="button"
+            onClick={close}
+          >
+            Cancel
+          </Button>
+          <ActionButton
+            appearance="positive"
+            className="u-no-margin--bottom"
+            loading={formik.isSubmitting}
+            disabled={
+              !formik.isValid || storagePoolsLoading || clusterMembersLoading
+            }
+            onClick={() => void formik.submitForm()}
+          >
+            Duplicate and start
+          </ActionButton>
+        </>
+      }
+    >
+      <Form onSubmit={formik.handleSubmit}>
+        <Input
+          {...formik.getFieldProps("instanceName")}
+          type="text"
+          label="New instance name"
+        />
+        {isClustered ?? (
```

Comment:
```suggestion
        {isClustered && (
```
Otherwise this never shows.